jscomp/super_errors/super_typecore.ml

```diff
@@ -131,9 +131,9 @@ let report_error env ppf = function
         | None ->
           super_report_unification_error ppf env trace
             (function ppf ->
-                fprintf ppf "This value has type:")
+                fprintf ppf "This has type:")
             (function ppf ->
-                fprintf ppf "But was expected to be:");
+                fprintf ppf "But somewhere wanted:");
```

Comment:
cc @jaredly. I took the occasion to revert the wording a bit because:

- I'm not sure it'll always be `value` when the reporting triggers. Maybe it is?
- "Was expected to be" sounds like the right type is the second one. I've originally used such phrasing, but decided against it, because it's very possible that the "expected" one is actually the wrong one.